packages/valory/skills/abstract_round_abci/tests/test_base_rounds.py

```diff
@@ -264,6 +275,28 @@ def test_run(
 
         self._test_payload_with_wrong_round_count(test_round)
 
+        test_round.done_event = "DONE_EVENT"
+        return_value = cast(Tuple[BaseSynchronizedData, Enum], test_round.end_block())
+        assert return_value[-1] == test_round.done_event
+
+        test_round.none_event = "NONE_EVENT"
```

Comment:
Don't use magic constants. We already have events declared elsewhere.

Reply:
imo we should introduce `DUMMY_EVENTS` in test_tools for tests because these should be enums and not strings. We have had issues with this earlier where tests used strings as payloads instead of actual payloads (which were dealt with differently in `collections.Counter`).